<ui version="4.0" >
 <class>TotRecalBase</class>
 <widget class="QDialog" name="TotRecalBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define relative ToT Chi/Clo calibration</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="OneForAll" >
       <property name="text" >
        <string>Use one calib factor for all chips</string>
       </property>
       <property name="checked" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="TextLabel1" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Global calibration:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="GlobalCal" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <spacer name="Spacer2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="ChipcalLabel" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Chip-by-chip calibration:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Q3Table" name="ChipCalib" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="numRows" >
            <number>16</number>
           </property>
           <property name="numCols" >
            <number>1</number>
           </property>
           <row>
            <property name="text" >
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>8</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>9</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>10</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>11</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>12</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>13</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>14</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>15</string>
            </property>
           </row>
           <row>
            <property name="text" >
             <string>16</string>
            </property>
           </row>
           <column>
            <property name="text" >
             <string>1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="PushButton5" >
           <property name="text" >
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="Spacer4" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="OKButton" >
         <property name="text" >
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FitButton" >
         <property name="text" >
          <string>Fit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>OneForAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>GlobalCal</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OneForAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>TextLabel1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OneForAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChipcalLabel</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OneForAll</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChipCalib</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
